src/main/resources/hudson/plugin/versioncolumn/Messages.properties

```diff
@@ -1,6 +1,6 @@
 VersionMonitor.DisplayName=Remoting Version
-VersionMonitor.OfflineCause=This node is offline because it uses an old slave.jar
-VersionMonitor.MarkedOffline=Making {0} offline temporarily due to the use of an old slave.jar
+VersionMonitor.OfflineCause=This node is offline because it uses an old remoting.jar
+VersionMonitor.MarkedOffline=Making {0} offline temporarily due to the use of an old remoting.jar
```

Comment:
Thanks for the pull request!

I believe it is actually called `agent.jar` rather than `remoting.jar`.  At least that's the name that I see in my agent definition page

```suggestion
VersionMonitor.OfflineCause=This node is offline because it uses an old agent.jar
VersionMonitor.MarkedOffline=Making {0} offline temporarily due to the use of an old agent.jar
```

![Screenshot 2022-04-28 195505](https://user-images.githubusercontent.com/156685/165874069-613d493e-d385-4a35-95d3-2bbad6927194.png)

Reply:
Worker nodes use remoting.<version>.jar and have a line in the config file like this, I have to update both:
<arguments>-Xrs -Djsse.enableSNIExtension=false -jar "%BASE%\remoting-4.13.jar" -jnlpUrl http://jenkins/computer/Android%20Studio%20(1)/slave-agent.jnlp -secret 1234567890abcdefghij1234567890abcdefghij</arguments>

Reply:
> Worker nodes use remoting.version.jar and have a line in the config file like this, I have to update both: `<arguments>-Xrs -Djsse.enableSNIExtension=false -jar "%BASE%\remoting-4.13.jar" -jnlpUrl http://jenkins/computer/Android%20Studio%20(1)/slave-agent.jnlp -secret 1234567890abcdefghij1234567890abcdefghij</arguments>`

I've never seen that content displayed to a Jenkins user.  I'm quite surprised that the remoting version number is included in the file name.  I'm accustomed to jar file references that do not include a version number in the name of the jar file.  Can you describe the case where Jenkins displays that file content to someone?

If it is not displayed to a Jenkins user, I think it is a mistake to use the word "remoting" when the word "agent" is used in pages that are displayed to Jenkins users.